app/assets/javascripts/discourse/lib/utilities.js.es6

```diff
@@ -660,5 +660,32 @@ export function postRNWebviewMessage(prop, value) {
   }
 }
 
+function reportToLogster(name, error) {
+  const data = {
+    message: `${name} theme/component is throwing errors`,
+    stacktrace: error.stack
+  };
+
+  Ember.$.ajax("/logs" + "/report_js_error", {
+    data,
+    type: "POST",
+    cache: false
+  });
+}
+// this function is used in lib/theme_javascript_compiler.rb
+export function rescueThemeError(name, error, api) {
+  console.error(`"${name}" error:`, error);
+  reportToLogster(name, error);
+  const currentUser = api.getCurrentUser();
```

Comment:
I would add a line break before and after this block. Doesn't have any direct link with console logging and the alert warning.